### PR TITLE
Update public IP API information for cloud

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**
- docs(cloud): add Public IP API
  - Created redirect for previous slug
- chore(misc): remove duplicate entry on package.json

image

### CODE REVIEW

**Review by bianpengyuan — APPROVED** [review]

**Comment by DanRoscigno** [comment]
Are there more APIs coming, and if so, is there a chance that they will not be related to security?  (I am wondering if APIs belong under Cloud > Security, or Cloud > manage?)

File: docs/en/cloud/security/ip-public-api.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      ],
+      "region": "ap-south-1",
+      "s3_endpoint": "vpce-0a975c9130d07276d"
+    },
+...
+```
+
+For example a ClickHouse cloud service running in `us-east-2` using the integration to connect to an RDS in AWS, should have the following Inbound security group rules:
+
+<img src={require("@site/docs/en/_snippets/images/aws-rds-mysql.png").default} style={{width:'70%', padding:'20px'}} alt="AWS Security group rules"/>
```
Comment (DanRoscigno) [inline docs/en/cloud/security/ip-public-api.md:45]:
```suggestion
![AWS Security group rules](@site/docs/en/_snippets/images/aws-rds-mysql.png) 
```

I can't read the image with those settings.

File: docs/en/cloud/security/ip-public-api.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+For the same ClickHouse cloud service running in `us-east-2`, but this time connected to an MySQL in GCP, the `Authorized networks` should look like this:
+
+<img src={require("@site/docs/en/_snippets/images/gcp-authorized-network.png").default} style={{width:'40%', padding:'20px'}} alt="GCP Authorized networks"/>
```
Comment (DanRoscigno) [inline docs/en/cloud/security/ip-public-api.md:49]:
```suggestion
![GCP Authorized networks](@site/docs/en/_snippets/images/gcp-authorized-network.png)
```

File: docs/en/cloud/security/ip-public-api.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## ClickHouse Cloud public API
+
+The ClickHouse Cloud public API endpoint is available on [`https://api.clickhouse.cloud/static-ips.json`](https://api.clickhouse.cloud/static-ips.json).
+
+This API provides public IPs and endpoints for ClickHouse services, such as ingress/egress IPs and AWS S3 endpoints per region and cloud.
+
+## Integrations could require egress access
+
+If you are using integration like the MySQL or PostgreSQL Engine, it is possible that you need to authorize ClickHouse Cloud to access to your instances.
```
Comment (DanRoscigno) [inline docs/en/cloud/security/ip-public-api.md:15]:
```suggestion
If you are using integration like the MySQL or PostgreSQL Engine, it is possible that you need to authorize ClickHouse Cloud to access your instances.
```

**Review by DanRoscigno — CHANGES_REQUESTED** [review]

File: docs/en/cloud/security/ip-public-api.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply docs/en/cloud/security/ip-public-api.md:45]: Interesting. It works here locally:

image

**Comment by GMartinez-Sisti** [comment]
> Maybe I'm confused by the title of this page. Is this our new "Cloud API", or is it some kind of special "IP API"? (I don't know what an IP API is...)

This is just a new way we have to share our Cloud Public IPs and Endpoints with our customers. We called "Public IP API" because it allows interaction/query/filtering from customer tooling. We don't have any preference on the naming, feel free to change it.

**Comment by marcelbirkner** [comment]
> Maybe I'm confused by the title of this page. Is this our new "Cloud API", or is it some kind of special "IP API"? (I don't know what an IP API is...)

https://api.clickhouse.cloud is the ClickHouse Cloud API that ControlPlane and DataPlane teams share.

This endpoint https://api.clickhouse.cloud/static-ips.json is providing Ingress/Egress IPs from our CH Cloud infrastructure. This is provided by dataplane team.

All other endpoints will be provided by ControlPlane team and will be used for the ControlPlane API. 

We are using the same domain to not confuse customers and have a single point of knowledge.

**Comment by marcelbirkner** [comment]
GitHub has a similar API endpoint and they call it [meta](https://api.github.com/meta) API endpoint

```
About GitHub's IP addresses
GitHub serves applications from multiple IP address ranges, which are available using the API.

You can retrieve a list of GitHub's IP addresses from the [meta](https://api.github.com/meta) API endpoint. For more information, see "[Meta](https://docs.github.com/en/rest/meta)."
```

**Comment by rfraposa** [comment]
> GitHub has a similar API endpoint and they call it [meta](https://api.github.com/meta) API endpoint

Do you think this endpoint could grow beyond IP addresses someday? "meta" seems cool, but I definitely do not want to be responsible for the name! 

If you like "Public IP API" then that's cool. Or maybe something like "Cloud Endpoints API" might be more clear?

**Comment by marcelbirkner** [comment]
I would vote for "Cloud Endpoints API" since it contains more information besides IP. 

@GMartinez-Sisti WDYT?

**Review by DanRoscigno — APPROVED** [review]

**Comment by GMartinez-Sisti** [comment]
Redirect from previous slug works:
```sh
→ curl -i https://clickhouse.com/docs/en/manage/security/ip-egress-traffic-list
HTTP/2 200
date: Wed, 05 Apr 2023 11:07:29 GMT
content-type: text/html; charset=utf-8
cf-ray: 7b3153e558da94ef-LIS
accept-ranges: bytes
access-control-allow-origin: *
age: 30
cache-control: public, max-age=0, must-revalidate
content-disposition: inline; filename="ip-egress-traffic-list"
strict-transport-security: max-age=0; includeSubDomains; preload
cf-cache-status: DYNAMIC
x-content-type-options: nosniff
x-vercel-cache: HIT
x-vercel-id: cdg1:cdg1::rzpvm-1680692849534-afc621f62172
server: cloudflare
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400

<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <meta http-equiv="refresh" content="0; url=/docs/en/manage/security/cloud-endpoints-api">
    <link rel="canonical" href="/docs/en/manage/security/cloud-endpoints-api" />
  </head>
  <script>
    window.location.href = '/docs/en/manage/security/cloud-endpoints-api' + window.location.search + window.location.hash;
  </script>
</html>
```

**Pull request merged**